src/groundlight/client.py

```diff
@@ -95,6 +106,20 @@ def __init__(self, endpoint: Optional[str] = None, api_token: Optional[str] = No
                     ),
                 ) from e
 
+        should_disable_tls = disable_tls_verification or os.environ.get(DISABLE_TLS_VARIABLE_NAME, None) is not None
+
+        if should_disable_tls:
```

Comment:
```suggestion
        if should_disable_tls_verification:
```
We're not really disabling TLS - just the verification.